app/endpoints.js

```diff
@@ -0,0 +1,24 @@
+import { identityServerUrl, organisationApiUrl, oidcBaseUri } from './config';
+
+const endpoints = {
+  // GET endpoints
+  getIdentityApiHealth: () => `${oidcBaseUri}/health/ready`,
+
+  getOrganisations: () => `${organisationApiUrl}/api/v1/Organisations`,
+  getOrgById: opts => `${organisationApiUrl}/api/v1/Organisations/${opts.organisationId}`,
+  getOrgByOdsCode: opts => `${organisationApiUrl}/api/v1/ods/${opts.odsCode}`,
+
+  getUsers: opts => `${identityServerUrl}/api/v1/Organisations/${opts.organisationId}/Users`,
+  getUserById: opts => `${identityServerUrl}/api/v1/Users/${opts.userId}`,
+
+  // POST endpoints
+  postAddOrg: () => `${organisationApiUrl}/api/v1/Organisations`,
+
+  postAddUser: opts => `${identityServerUrl}/api/v1/Organisations/${opts.organisationId}/Users`,
+  postUserStatus: opts => `${identityServerUrl}/api/v1/Users/${opts.userId}/${opts.status}`,
+
+  // PUT endpoins
+  putUpdateOrganisation: opts => `${organisationApiUrl}/api/v1/Organisations/${opts.organisationId}`,
+};
+
+export const getEndpoint = ({ endpointLocator, options }) => endpoints[endpointLocator](options);
```

Comment:
Should this go into the library?

Reply:
that would mean sending all the endpoints into the librry?

Reply:
I mean just the getEndpoint function

Reply:
but we would need to pass all the endpoints into the getEndpoint function

Reply:
You're right I'm being stupid here, was thinking it worked differently to how it does